module-telemetry/src/main/java/com/mapbox/maps/module/telemetry/MapTelemetryImpl.kt

```diff
@@ -17,7 +22,8 @@ class MapTelemetryImpl : MapTelemetry {
 
   private val appContext: Context
   private val accessToken: String
-  private val telemetry: MapboxTelemetry
+  private val eventsService: EventsServiceInterface
+  private val telemetryService: TelemetryService
```

Comment:
nit: consider adding a comment, something like `// We need telemetryService for location and metrics event which will be sent automatically if telemetryService is initialized.`